events/events.go

```diff
@@ -0,0 +1,37 @@
+package events
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Broker of events
+type Broker interface {
+	Publish(topic string, opts ...PublishOption) error
+	Subscribe(topic string, opts ...SubscribeOption) (<-chan Event, error)
+}
+
+// Store of events
+type Store interface {
```

Comment:
Events Store or Events Stream?

Reply:
I think store is more descriptive, Stream indicates something you subscribe to and I think that's more of the brokers job.

Reply:
+1 for store from my side

Reply:
If this is timeseries like moreso than event streaming then I agree. If its more kafka like then its event streaming. Which is it?

Reply:
Or do we say that events.Store and events.Stream are two separate things that can coexist here?